nemo/export/trt_llm/tensorrt_llm_build.py

```diff
@@ -371,6 +372,12 @@ def build_and_save_engine(
     lora_target_modules=None,
     max_prompt_embedding_table_size=0,
     enable_multi_block_mode: bool = False,
+    paged_kv_cache: bool = True,
+    remove_input_padding: bool = True,
+    max_num_tokens: int = None,
+    opt_num_tokens: int = None,
```

Comment:
Formally `max_num_tokens` and `opt_num_tokens` parameters should have `Optional[int]` typing